frontend/src/app/components/wp-query/url-params-helper.ts

```diff
@@ -86,6 +86,10 @@ export class UrlParamsHelperService {
       paramsData.hl = query.highlightingMode;
     }
 
+    if (query.highlightingMode === 'inline' && query.highlightedAttributes) {
+      paramsData.hla = query.highlightedAttributes.map(el => el.href);
```

Comment:
@oliverguenther Do we really want to use the `href` as attribute identifiers?

Reply:
Yes, they are the HAL identifiers of the resource. However We can map it here to its ID and unwrap it to `/api/v3/columns/ID` again in the URLParamsHelper when sending to the query endpoint, since this point here is used for the URL query string and should be concise